app/src/main/java/org/schabi/newpipe/util/DrawableResolver.kt

```diff
@@ -2,25 +2,24 @@ package org.schabi.newpipe.util
 
 import android.content.Context
 import android.graphics.drawable.Drawable
+import android.util.TypedValue
 import androidx.annotation.AttrRes
 
 /**
  * Utility class for resolving [Drawables](Drawable)
  */
-class DrawableResolver {
-    companion object {
-        @JvmStatic
-        fun resolveDrawable(context: Context, @AttrRes attrResId: Int): Drawable? {
-            return androidx.core.content.ContextCompat.getDrawable(
-                context,
-                android.util.TypedValue().apply {
-                    context.theme.resolveAttribute(
-                        attrResId,
-                        this,
-                        true
-                    )
-                }.resourceId
-            )
-        }
+object DrawableResolver {
+    @JvmStatic
+    fun resolveDrawable(context: Context, @AttrRes attrResId: Int): Drawable? {
+        return androidx.core.content.ContextCompat.getDrawable(
```

Comment:
Thanks for fixing! However, it seems you missed a spot :P

Reply:
Thx for the hint.
I really shouldn't blindly copy past code from stackoverflow (or something like this) next time 😆